opta/layer.py

```diff
@@ -75,6 +35,55 @@
 from opta.utils import deep_merge, hydrate, logger, yaml
 from opta.utils.dependencies import validate_installed_path_executables
 
+PROCESSOR_DICT: Dict[str, str] = {
```

Comment:
Can you please add a unit test that checks that all of the processors in this dict exist? Something like:
```python
for processor in PROCESSOR_DICT.values():
    pymodule_path = generate_pymodule_path(processor)
    importlib.import_module(pymodule_path)
```

Reply:
Added a test to check if each module in the dictionary is actually in the directories.